config/21_regions_EU11/scenario_config_21_EU11_ARIADNE.csv

```diff
@@ -1,8 +1,8 @@
 title;start;CES_parameters;c_keep_iteration_gdxes;cm_startIter_EDGET;cm_calibration_string;optimization;c_testOneRegi_region;cm_nash_mode;subsidizeLearning;c_regi_earlyreti_rate;c_tech_earlyreti_rate;trade;cm_import_EU;c_ccsinjecratescen;cm_bioprod_histlim;cm_BioImportTax_EU;cm_biotrade_phaseout;cm_1stgen_phaseout;c_BaselineAgriEmiRed;cm_altFeEmiFac;cm_ariadne_VRECapFac_adj;cm_wind_offshore;cm_VRE_supply_assumptions;cm_flex_tax;cm_PriceDurSlope_elh2;cm_build_H2costAddH2Inv;cm_CESMkup_build;cm_CESMkup_ind;cm_FEtax_trajectory_abs;cm_FEtax_trajectory_rel;cm_INNOPATHS_inco0Factor;cm_INNOPATHS_adj_coeff_multiplier;cm_INNOPATHS_adj_seed_multiplier;cm_iteration_max;regionmapping;cm_techcosts;cm_rcp_scen;cm_iterative_target_adj;c_budgetCO2;carbonprice;cm_co2_tax_2020;cm_peakBudgYr;cm_taxCO2inc_after_peakBudgYr;cm_CO2priceRegConvEndYr;regipol;cm_quantity_regiCO2target;cm_regiCO2target;cm_implicitFE;cm_implFETarget;cm_NucRegiPol;cm_CoalRegiPol;capitalMarket;industry;buildings;transport;cm_EDGEtr_scen;cm_co2_tax_growth;cm_emiscen;techpol;cm_multigasscen;cm_INCONV_PENALTY_bioSwitch;cm_POPscen;cm_GDPscen;cm_demScen;cm_startyear;path_gdx;path_gdx_ref
 _____Calibration_____;0;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-Calib_EU;0;calibrate;0;14;off;nash;DEU;parallel;off;GLO 0.09, EUR_regi 0.15;GLO.pc 0.05, EUR_regi.pc 0.15;se_trade;off;1;-1;0;0;0;0;EUR_regi, NEU_regi;on;1;1;0;20;0.2;standard;standard;off;off;off;off;off;1;./config/21_regions_EU11/regionmapping_21_EU11.csv;REG;none;0;0;none;-1;2100;3;2050;regiCarbonPrice;off;off;off;off;on;on;debt_limit;subsectors;simple;edge_esm;Mix;1.05;1;none;2;on;pop_SSP2EU;gdp_SSP2EU;gdp_SSP2EU;2005;;
+Calib_EU;0;calibrate;0;14;off;nash;DEU;parallel;off;GLO 0.09, EUR_regi 0.15;;EUR_regi.pc 0.15;se_trade;off;1;-1;0;0;0;0;EUR_regi, NEU_regi;on;1;1;0;20;0.2;standard;standard;off;off;off;off;off;1;./config/21_regions_EU11/regionmapping_21_EU11.csv;REG;none;0;0;none;-1;2100;3;2050;regiCarbonPrice;off;off;off;off;on;on;debt_limit;subsectors;simple;edge_esm;Mix;1.05;1;none;2;on;pop_SSP2EU;gdp_SSP2EU;gdp_SSP2EU;2005;;
```

Comment:
You don't need a tech specific value (EUR_regi.pc 0.15) in the calib and base scenarios after your changes, as all techs will use 0.15 anyway in this case, no?

Reply:
Ah, but the double semicolon there is a bug.

Reply:
I would definitely prefer to remove it if it is unnecessary

Reply:
OK, do you have a convention for a null value for these types of switches? Will it accept a simple string, e.g. "none"?

Reply:
I usually use "off" as a string to disable using the switch. You can check the regipol module to see examples of that.

Reply:
Thanks, @Renato-Rodrigues! This has been fixed